doc/manuals/admin/database_model.md

```diff
@@ -123,7 +123,7 @@ Regarding `location.coords` in can use several formats:
   "fixed" cases.
 
 Note that coordinate pairs use the longitude-latitude order, which is opposite to the order used
-in the [Simple Location Format](../orion-api.md#simple-location-format). This is due to the internal
+in the [Geographical Queries](../orion-api.md#geographica-queries). This is due to the internal
 [MongoDB geolocation implementation](http://docs.mongodb.org/manual/tutorial/query-a-2dsphere-index/),
 (which is based in GeoJSON) uses longitude-latitude order. However, other systems closer
 to users (e.g. GoogleMaps) use latitude-longitude format, so we have used the latter for the API.
```

Comment:
```suggestion
to users (e.g. GoogleMaps) use latitude-longitude format, so we have used the latter for the Geographical Queries API.
```

Reply:
Fixed in [5371cd6](https://github.com/telefonicaid/fiware-orion/pull/4367/commits/5371cd667ea71a5e6515c43464db2045e453f0ad)